ui/mirage/config.js

```diff
@@ -22,11 +22,280 @@ export default function () {
     };
   });
 
-  this.get('/sys/internal/ui/feature-flags', (db) => {
-    const featuresResponse = db.features.first();
+  this.get('/sys/internal/ui/feature-flags', () => {
```

Comment:
I know this is just a mirage file, but this doesn't look like the right path for this response?

Reply:
Good catch! This was a merge conflict that had been resolved incorrectly when main was last merged into this branch